### PR TITLE
Desktop responsiveness

### PR DESCRIPTION
**Pull request by Jolak5**
In this PR, I

1. added the media query to build the desktop version
2. created a nav bar for desktop that is hidden on mobile
3. rearranged the layout of the website using flex and grid

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,47 +13,54 @@
...
+        <div>
+          <h1>
+            Hey there. I’m Joseph <br />
+            <span class="header-span"> I’m a software developer</span>
+          </h1>
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
```
Comment (ha-manel) [inline index.html:40]: - [ ] Great job on making this look awesome, however, it's looking huge in the mobile version. Kindly decrease the `font-size` in mobile view 👍

## Your design:
image

## Figma design:
 
image

File: index.html [hunk 2]
```diff
@@ -77,6 +84,30 @@ <h4>Multi-Post Stories <br />Gain+Glory</h4>
...
+        <div class="card">
+          <div class="bottom-card">
+            <h4>Multi-Post Stories <br />Gain+Glory</h4>
+            <ul>
+              <div class="top-card"></div>
+              <li>Html</li>
+              <li>Css</li>
+              <li>Javascript</li>
+            </ul>
+            <button class="card-button">See Projects</button>
```
Comment (ha-manel) [inline index.html:96]: - [ ] Kindly add all button interactions as shown in Figma 👍 

image

File: index.html [hunk 3]
```diff
@@ -77,6 +84,30 @@ <h4>Multi-Post Stories <br />Gain+Glory</h4>
...
+          </div>
+        </div>
```
Comment (ha-manel) [inline index.html:110]: - [x] Kindly add 2 more cards to match the design 👍 

## Your design:
image

## Figma design:
image

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @Jolak5 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️

## Required Changes:
_Kindly check the comments below_


------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Jolak5) [reply index.html:110]: Hi, thank you for your review. I have implemented the orders. However, I have more than one card in this section. Can you kindly check please?

**Review by aimalamiri — APPROVED** [review]
Hi @Jolak5, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

ApprovedDavieGIF


## Highlights

- The desktop version is designed very well and looks fantastic ✔️ ⭐
- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**